.github/workflows/holochain-build-and-test.yml

```diff
@@ -44,7 +36,7 @@ jobs:
                 - build-release-automation-tests
                 - build-release-automation-tests-repo
                 - build-holochain-tests-static-all
-              extra_arg: "--override-input versions ./versions/0_2 --override-input holochain ${{ inputs.repo_path }}"
+              extra_arg: "--override-input versions ./versions/weekly --override-input holochain ${{ inputs.repo_path }}"
```

Comment:
This change wasn't actually intentional but isn't this correct? For verification we want to start from close to latest